### ISSUE TITLE
install sessions/redis by dep has a conflict in packages

### ISSUE THREAD

**Issue by axiaoxin** [gin-contrib/sessions#83]
using dep add sessions/redis, then panic:

    ../vendor/github.com/gin-contrib/sessions/redis/redis.go:53:42: 
     cannot use pool (type *"../vendor/github.com/gomodule/redigo/redis".Pool) as type 
     *"../vendor/github.com/garyburd/redigo/redis".Pool in argument to redistore.NewRediStoreWithPool

the `github.com/boj/redistore/redistore.go:18` import `github.com/garyburd/redigo/redis`

**Comment by wushaobo** [comment]
Hi, has there been a schedule for fixing this bug and release?

**Comment by wushaobo** [comment]
Hi, has there been a schedule for fixing this bug and release?

**Comment by delphinus** [comment]
I think your `boj/redistore` is too old. The current build uses `gomodule/redigo`, but your revision seems to use `garyburd/redigo` (old repo) instead of that.

**Comment by wushaobo** [comment]
@delphinus  You are right. Thank you. The `boj/redistore` is installed automatically by dep using its released version `v1.2` rather than the newer version specified by `gin-contrib/sessions`.  I think it's time to migrate to go mod from dep.
